### ISSUE TITLE
Facilitate networking with an after-conf drink

### ISSUE THREAD

**Issue by fabricehong** [swissmobidevs/appbuilders16#20]
Nothing better than offering a beer at the end of the first day (the second day, people are going back and cannot stay late) to ease people mingling .

Creating a space for networking is important. For some people it's an major component about going to conferences.

**Comment by BalestraPatrick** [comment]
Good idea! We wanted to organize an event on Monday evening but we didn't have time in the end. It was too late to book anything. We will definitely to that in the future 👍

**Comment by bontoJR** [comment]
Well, this was really planned, but the surrounding places were or:
1. already booked
2. freaking expensive
3. too small to fit ~250 people (200 attendees, 32 speakers, plus 10 member of the staff)

definitely something that will be fixed next year.
